Libraries/Template/Components/Header.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```

Comment:
I saw you've added flow config. Do you plan to add type definition for those files?

Reply:
I'm not entirely familiar with flow sir so I don't have a plan for flow

Reply:
base on what we have in the current `[App.js](https://github.com/facebook/react-native/blob/master/template/App.js)`, and since Flow isn't required to run RN, I think it makes sense to not have  Flow nor propTypes.

Reply:
I agree w/ you @elucaswork